src/main/twirl/uk/gov/hmrc/play/views/layouts/trusted_helper_banner.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+* Copyright 2015 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@(name: Option[String], returnLinkUrl : String, regime: TrustedHelperRegime)
+
+<div class=" grid-wrapper attorneyContainer">
```

Comment:
how is this file different from attorney_banner? Shouldn't we have just one, with this new interface, perhaps defaulting to standAlone TrustedHelperRegime ?